integration-tests/benchmark/keeper_test.go

```diff
@@ -383,20 +388,24 @@ func SetupAutomationBenchmarkEnv(t *testing.T, keeperTestConfig types.KeeperBenc
 						},
 					},
 					"geth": map[string]interface{}{
-						"blocktime": blockTime,
+						"blocktime":      blockTime,
+						"capacity":       "20Gi",
+						"startGaslimit":  "20000000",
+						"targetGasLimit": "30000000",
 					},
 				},
 			}))
 	}
 
+	// TODO we need to update the image in CTF, the old one is not available anymore
```

Comment:
can we add a ticket to track this?

Reply:
does this affect current test?

Reply:
This does not affect the test. This only adds a blockscout explorer for the test which we can use to debug txns if need be.

Reply:
@Tofel when I ran this test last week (from develop), I could see that it was able to get the older v1 blockscout image from ECR and that seemed to work. Maybe it can be left as it was before. Once the chart for v2 is there, it can be updated separately.

Reply:
@anirudhwarrier for me it was failing, because blockscout image could not be fetched. Maybe we have updated the image we use in CTF and develop uses that CTF version?